test/python/test_statevector_simulator_projectq.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+
+# Copyright 2018 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+
+import unittest
+from qiskit import qasm, unroll, QuantumProgram, QuantumJob
+from qiskit.backends.local.statevector_simulator_projectq import StatevectorSimulatorProjectQ
+from test.python.common import QiskitTestCase
+
+
+class StatevectorSimulatorProjectQTest(QiskitTestCase):
```

Comment:
Can you use a similar mechanism as [the one in `test_qasm_simulator_projectq.py`](https://github.com/QISKit/qiskit-sdk-py/blob/a3cab9b9db49ddcca8e6f4e02ee672ede25b1d3f/test/python/test_qasm_simulator_projectq.py#L35-L43) for making this TestCase to be skipped if ProjectQ is not present? Since ProjectQ is an optional dependency, in some case the test suite is run without it installed, and [result in a failure](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/373478295#L1225).

Reply:
Done.

Reply:
Sorry for not mentioning it explicitly - but can you also do something similar for `test_statevector_simulator_cpp`, instantiating `StatevectorSimulatorCpp()` in the `try` block? The CI build seems to be failing [because of that](https://travis-ci.org/QISKit/qiskit-sdk-py/jobs/374274269#L1225).

Reply:
Sure, done.